onlineweb4/settings/gsuite.py

```diff
@@ -49,6 +49,7 @@
         'itex' : 'itex',
         'velkom': 'velkom',
         'interessegrupper': 'interessegrupper',
-        'online-il': 'online-il'
+        'online-il': 'oil',
```

Comment:
Didn't this work when it was `online-il`?

Reply:
Yeah, sorry, it was a formal fallacy from my side. Fixed it in last update.

Reply:
Btw as a general remark, it would be great to have more descriptive commit-messages. "Update gsuite.py" doesn't really tell me anything, it would be better to say what and *why* the change was made.